setup.py

```diff
@@ -10,6 +10,7 @@
       long_description="",
       url="",
       py_modules=['opencivicdata.apps', 'opencivicdata.common'],
+      namespace_packages=['opencivicdata'],
```

Comment:
same as other namespace comment above, if we can demonstrate it is fully backwards/forwards compatible I'd be OK making an exception here